### PR TITLE
Cli: Implement sweep address call

### PR DESCRIPTION
**Pull request by TheCharlatan**
This can be used to salvage funds from funding addresses in case something goes wrong.

### CODE REVIEW

**Review by zkao — APPROVED** [review]
good PR, minor comments only

File: src/cli/command.rs [hunk 1]
```diff
@@ -252,6 +254,29 @@ impl Exec for Command {
...
+                    runtime.request(
+                        ServiceId::Farcasterd,
+                        Request::SweepBitcoinAddress(SweepBitcoinAddress {
+                            source_address,
+                            source_private_key: secret_key,
+                            destination_address,
+                        }),
+                    )?;
+                    runtime.report_response_or_fail()?;
+                }
```
[the remark this answers is not in the document]
Comment (TheCharlatan) [reply src/cli/command.rs]: Done: 4c26641355db0fc3380c011915065d354c1e5afd

File: src/farcasterd/runtime.rs [hunk 2]
```diff
@@ -1602,6 +1620,116 @@ impl Runtime {
...
+                        "--network".to_string(),
+                        network.to_string(),
+                    ];
+                    args.append(&mut syncer_servers_args(&self.config, coin, network)?);
+                    info!("launching syncer with: {:?}", args);
+                    launch("syncerd", args)?;
+                    self.spawning_services.insert(s, ServiceId::Farcasterd);
+                    if let Some(xs) = self.syncer_clients.get_mut(&k) {
+                        xs.insert(source.clone());
+                    }
```
[the remark this answers is not in the document]
Comment (TheCharlatan) [reply src/farcasterd/runtime.rs]: Yes, good catch, patched in d5fb5afc29abb9d8a3813fe5b5fb59cdbb417e72

File: src/farcasterd/runtime.rs [hunk 3]
```diff
@@ -1602,6 +1620,116 @@ impl Runtime {
...
+                    endpoints.send_to(
+                        ServiceBus::Ctl,
+                        ServiceId::Farcasterd,
+                        ServiceId::Syncer(coin, network),
+                        request,
+                    )?;
+                }
+            }
+
+            Request::SyncerEvent(Event::SweepSuccess(success)) => {
```
[the remark this answers is not in the document]
Comment (TheCharlatan) [reply src/farcasterd/runtime.rs]: Done in: 03e02ad4f27818cfabfd5e53237eb0435e723454

**Pull request merged**